packages/vite/src/waku-lib/build-server.ts

```diff
@@ -76,8 +76,10 @@ export async function serverBuild(
               code += '"use client";'
             }
 
-            const serverKeys = Object.keys(serverEntryFiles)
-            if (chunk.moduleIds.some((id) => serverKeys.includes(id))) {
+            const serverValues = Object.values(serverEntryFiles)
```

Comment:
`Object.values` instead of `Object.keys` was the solution to getting RSFs working